sample/build.gradle.kts

```diff
@@ -0,0 +1,43 @@
+plugins {
+    alias(libs.plugins.android.application)
+    alias(libs.plugins.kotlin.android)
+    alias(libs.plugins.secrets.gradle.plugin)
+}
+
+android {
+    namespace = "com.example"
+
+    compileSdk = libs.versions.exampleCompileSdk.get().toInt()
+
+    buildFeatures {
+        viewBinding = true
```

Comment:
Food for though: I think at this stage its safe to abandon the old View-system. Compose has been around for 3 years now.